src/services/createRouterHooks.ts

```diff
@@ -142,19 +98,18 @@ export function createRouterHooks(): RouterHooks {
     }
   }
 
-  async function runAfterRouteHooks({ to, from }: HookContext): Promise<AfterRouteHookResponse> {
-    const { global, component } = store
-    const routeHooks = getAfterRouteHooksFromRoutes(to, from)
-    const globalHooks = getGlobalAfterRouteHooks(to, from, global)
+  const runAfterRouteHooks: AfterHookRunner = async (to, from) => {
+    const routeHooks = getAfterHooksFromRoutes(to, from)
+    const globalHooks = getGlobalAfterHooks(to, from, globalStore)
 
-    const allHooks: RouterAfterRouteHook[] = [
-      ...component.onAfterRouteLeave,
+    const allHooks: AfterHook[] = [
+      ...componentStore.onAfterRouteLeave,
```

Comment:
Seems like this should be named "componentHooks" since the others are "globalHooks" and "routeHooks"